src/audio/Kconfig

```diff
@@ -73,6 +73,12 @@ config COMP_SEL
 	help
 	  Select for SEL component
 
+config COMP_DCBLOCK
+	bool "DC Blocking Filter component"
+	default y
+	help
+	  Select for DC Blocking Filter component
```

Comment:
This description could be more verbose, something that a generic firmware engineer understands.